### PR TITLE
Refactored mood color to reflect exact mental thresholds

### PR DESCRIPTION
**Pull request by n-fisher**
- Makes pawn tile colors accurate to individuals' thresholds
- Fixes 1.0 accessibility for anyone who didn't upgrade
--------
In recent versions of RW the mood intervals are based on `Verse.AI.MentalBreaker` constant float multipliers off of an individual's dynamic break threshold, rather than a def-determined static value. This PR has the background colors accurately reflect each individuals' break risk factor, by targeting `MentalBreaker` accessors.

Also, fixed your 1.0 support as I saw there were comments about that on Steam.

--------

All changes have been QAed on their respective RW versions

### CODE REVIEW

File: Source/ColoredMoodBar/HarmonyPatch.cs [hunk 1]
```diff
@@ -153,36 +154,22 @@ public class MoodPatch {
...
+            if (pawnState.BreakExtremeIsImminent) {
+                return Main.extremeBreakTex;
+            } else if (pawnState.BreakMajorIsImminent) {
+                return Main.majorBreakTex;
+            } else if (pawnState.BreakMinorIsImminent) {
+                return Main.minorBreakTex;
+            } else if (pawnState.CurMood < .65f) {
+                return Main.neutralTex;
+            } else if (pawnState.CurMood < .9f) {
+                return Main.contentTex;
```
Comment (bcooper94) [inline Source/ColoredMoodBar/HarmonyPatch.cs:169]: I can't remember why I had both a neutral and content color tbh. Was there any significance to the difference? I just see the different mental break thresholds and the >90% threshold for a chance of inspirations on https://www.rimworldwiki.com/wiki/Mood.

File: About/About.xml [hunk 2]
```diff
@@ -5,6 +5,7 @@
   <author>PorqueNoLosDos</author>
   <packageId>PorqueNoLosDos.ColorCodedMoodBar</packageId>
   <supportedVersions>
+    <li>1.0</li>
```
Comment (bcooper94) [inline About/About.xml]: I remember causing issues with 1.1 if users activated my mod without also having Harmony installed. It'd crash the game and clear their mod list, so I think I'd rather not support it directly unless that's fixed.
```suggestion
```

File: LoadFolders.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<loadFolders>
```
Comment (bcooper94) [inline LoadFolders.xml:1]: The game seems to work fine without this file even in 1.2. I wonder if adding this without a 1.2 definition will break 1.2 though.

File: Source/ColoredMoodBar/HarmonyPatch.cs (hunk 1 above)
Comment (n-fisher) [reply Source/ColoredMoodBar/HarmonyPatch.cs:169]: Yeah that’s the only useful reason. Splitting them is nice IMO so you can tell which pawns can pick up inspos and don’t need any mood boosts vs which ones are just neutral and are candidates for mood influencers

File: About/About.xml (hunk 2 above)
Comment (n-fisher) [reply About/About.xml]: Fine by me. Doubtful many people even use 1.0 any more

File: LoadFolders.xml (hunk 3 above)
Comment (n-fisher) [reply LoadFolders.xml:1]: If a version doesn’t have a matching loadFolder entry, it’ll default to the most recent version that does have an entry. So a 1.2 update would use the 1.1 loadFolder by default

**Pull request closed without merging**